js/scripts/dimension/playground_dimension.js

```diff
@@ -89,13 +88,24 @@ const loadPlaygroundDimension = () => {
         
         document.body.addEventListener('click', playMusic);
 
+        // NOTE: we can't activate music until the user has interacted with the canvas. (this issue is inherent to HTML5)
+        //  If listening for a click is the only way to activate music, that's fine. 
+        //  Our game's START button in the final version can be the trigger.
+        let playMusic = () => {
+            ASSET_MGR.playAudio("./music/starting_off_2_sample.wav", 0.1, true);
+        
+            // delete the event listener so that the music doesn't restart when the user clicks again
+            document.body.removeEventListener('click', playMusic);
+        };
+        
+        document.body.addEventListener('click', playMusic);
```

Comment:
Should it play music when you click @NathanHinthorne ? Idk maybe we will want to change it later but not a big deal. Will it trigger every time you shoot?